packages/firestore/src/local/mutation_queue.ts

```diff
@@ -102,6 +103,17 @@ export interface MutationQueue extends GarbageSource {
     batchId: BatchId
   ): PersistencePromise<MutationBatch | null>;
 
+  /**
+   * Returns the document keys for the mutation batch with the given batchId.
+   * This method returns `null` if the mutation has been removed.
```

Comment:
This isn't exactly accurate right? Since the cache intentionally persists after the mutation has been removed...  Perhaps something like:

This method returns `null` after the primary client has called removeMutationBatches() or a secondary client has called `removeCachedMutationKeys()`

Reply:
Reworded to:

```
Returns the document keys for the mutation batch with the given batchId.
For primary clients, this method returns `null` after
`removeMutationBatches()` has been called. Secondary clients return a
cached result until `removeCachedMutationKeys()` is invoked.
```